frappe/limits.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import unicode_literals
+import frappe
+import subprocess, os
+from frappe.model.document import Document
+from frappe.core.doctype.user.user import get_total_users
+from frappe.utils import flt, cint, now_datetime, getdate, get_site_path
+from frappe.utils.file_manager import MaxFileSizeReachedError
+from frappe.utils.data import formatdate
+from frappe import _
+
+class SiteExpiredError(frappe.ValidationError):
+	pass
+
+
+def has_expired():
+	if not frappe.conf.get("stop_on_expiry") or frappe.session.user=="Administrator":
```

Comment:
Hmm.. let's assume that if expiry is mentioned in limits, we want to stop on expiry and just use the limits system. We can remove the `stop_on_expiry` flag.